Dockerfile

```diff
@@ -0,0 +1,18 @@
+# Install python as a parent image 
+FROM python:3.7
+
+# Set working directory to app
+WORKDIR /app
+
+# Copy the current directory contents into the container at /app
+COPY . /app
+
+# Install any needed packages specified in requirements.txt
+# TODO: potentially remove 
```

Comment:
Agreed, remove- done by the makefile

Reply:
this is a good point @jcraggs, however in this case i'd probably opt for keeping the install commands as they are in the dockerfile instead of running `make install`  as it is clearer and more explicit.

the dockerfile is the production code whereas the makefile is currently just for convenience during development. for example, the way we install and run in the container may be different to how we install and run during development (e.g. in dev we may want to have linters installed and to run in debug mode).